import-service/importFileParser.js

```diff
@@ -0,0 +1,39 @@
+const AWS = require("aws-sdk")
+const s3 = new AWS.S3()
+require("../config")
+const csv = require("csv-parser")
+
+module.exports.importFileParser = async (event) => {
+  const params = {
+    Bucket: process.env.S3_BUCKET,
+    Key: `uploaded/products.csv`,
```

Comment:
You should get a file name from the event

Reply:
Noted. I will make these changes in another PR.